pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -406,6 +406,24 @@ type MonitoringSpec struct {
 	// Interval determines prometheus scrape interval
 	// +optional
 	Interval *metav1.Duration `json:"interval,omitempty"`
+
+	// Ceph exporter configuration
+	// +optional
+	Exporter *CephExporterSpec `json:"exporter,omitempty"`
+}
+
+type CephExporterSpec struct {
+	// Only performance counters greater than or equal to this option are fetched
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=10
+	// +kubebuilder:default=5
+	PerfCountersPrioLimit uint `json:"perfCountersPrioLimit,omitempty"`
+
+	// Time to wait before sending requests again to exporter server (seconds)
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=600
+	// +kubebuilder:default=5
+	StatsDelaySeconds uint `json:"statsDelaySeconds,omitempty"`
```

Comment:
This parses into an int64 in Ceph with no min/max behavior applied as far as I can tell. If we aren't 100% certain about the min/max values. I think it would be best to use the same type as Ceph and not set min/max.